docs/loop-3/add-pump.md

```diff
@@ -356,6 +357,67 @@ DanaKit doesn't provide a heartbeat by default.
+    The first 3 digits should always match, the patch always ends with 0, while the pump base always ends with 1.
+    Do not use the patch if they do not match.
+
+    ![Medtrum REF](img/medtrum-ref.JPG){width="450"}
+    {align="center"}
+
+You can only add the Medtrum patch pump when no pump is selected.
+If you already have a pump selected, you must first delete the pump as detailed in [Change Pump Type](#change-pump-type).
+
+1. Select [Insulin Type](#insulin-type)
+1. Check [Patch settings](#patch-settings)
+1. Enter your [pump base Serial Number](#check-pump-base-serial-number)
+1. Prime & Activate patch. See [Activation flow patch](#activation-flow)
+
+![Full Medtrum flow](img/medtrum-total.png){align="center"}
+
+### Patch settings
+
+!!! info "Good to know"
+    All of these settings can be adjusted throughout the lifespan of the patch. You can safely update them after activating the patch
+
+There are several settings you can setup while using Medtrum nano with the *Loop* app:
+
+1. `Max hourly insulin` & `Max daily insulin`: Medtrum Nano does not work with max bolus and max basal settings, it uses max hourly & max daily insulin. Just like the names suggests, it controls the maximum amount of insulin per hour or per day. 
+1. `Alarm setting`: The Medtrum Nano has the ability to make a beep it there is an occlusion, patch fault, empty battery, etc. These alarms can also be silenced using this setting
+1. `Patch lifetime`: Normally, the medtrum nano runs for 3 days and 8 hours. This ensures the funcioning of the patch. You can disable this limit with this setting to extend the lifetime of the patch untill an failure occures, like an occlusion, empty battery, etc
+1. `Notification for expiration patch`: If the `Patch lifetime` setting is set to normal lifetime, you have the ability to update this setting. With this setting, you can control at what point you get a reminder notification for replacing the patch
+
+### Check pump base serial number
+
+After checking the patch settings, you are prompted to enter the pump base serial number.
+This serial number is used by the *Loop* app to connect to the correct pump base.
+To see you serial number, grab your pump base and look at the bottom of the base.
+It should have a QR code and a small 8 character serial number.
+See image below:
+
+![Medtrum SN example](img/medtrum-sn.png){width="450"}
+{align="center"}
+
+### Activation flow
+
+!!! warning "IMPORTANT"
+    While the priming is running, DO NOT USE THE CANCEL BUTTON.
+    This might corrupt the activation flow, so please be patience while the priming process is running.
+
+After the serial number prompt, you need to setup the patch itself.
+Follow the visual guide in the *Loop* app and press "Start priming" to start priming the cannula of the patch.
+It is important to not attach the patch on your body while the priming process has not completed.
```

Comment:
suggest: 
"It is important to not attach the patch on your body ~~while~~ before the priming process completes. ~~has not completed.~~"